client/lib/analytics/ad-tracking/google-analytics.js

```diff
@@ -41,7 +41,8 @@ export function fireGoogleAnalyticsPageView(
 	urlPath,
 	pageTitle,
 	useJetpackGoogleAnalytics = false,
-	useAkismetGoogleAnalytics = false
+	useAkismetGoogleAnalytics = false,
+	useA4AGoogleAnalytics = false
```

Comment:
Should I be changing `fireGoogleAnalyticsPageView` function for us to track page view?

Reply:
yes I believe that is correct